Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -130,23 +130,23 @@ void GeomUtils::extrude(
 		std::cout << "extrude: face_loops_input.size() == 0" << std::endl;
 		return;
 		/*
-		throw oip::InconsistentModellingException(face_loops_input, "face_loops_input.size() == 0");
+		throw oip::InconsistentModellingException("face_loops_input.size() == 0");
 		*/
 	}
 	if( poly_data->points.size() > 0 )
 	{
 		std::cout << "extrude: points vec should be empty" << std::endl;
 		return;
 		/*
-		throw oip::InconsistentModellingException(poly_data, "points vec should be empty");
+		throw oip::InconsistentModellingException("points vec should be empty");
 		*/

```

Comment:
All throwing code is still commented out, and the legacy code wasn't replaced.

Reply:
Resolved: 3353249